### ISSUE TITLE
labelText Prop as PropType.node?

### ISSUE THREAD

**Issue by gtwilliams03** [Julusian/react-bootstrap-switch#49]
I am using some mini switches and it would be nice to put an html element in the labelText prop.

Like this:

```
<Switch onText='Yes' offText='No' bsSize='mini' 
    labelText={<FontAwesome name='link' tag={'i'} />} value={show} 
    onChange={(el, checkedState) => this.setState({ show: checkedState })} />
```
It actually renders quite nicely:

![image](https://user-images.githubusercontent.com/1564628/32931554-c467b62c-cb32-11e7-8384-203c5f431852.png)

But throws an error:

```
Warning: Failed prop type: Invalid prop `labelText` of type `object` supplied to `Switch`, expected `string`.
```

**Comment by Julusian** [comment]
Fixed in v15.5.3

**Comment by gtwilliams03** [comment]
Thank you!
